corehq/apps/analytics/tasks.py

```diff
@@ -595,3 +597,22 @@ def get_ab_test_properties(user):
         'a_b_test_variable_first_submission':
             'A' if deterministic_random(user.username + 'a_b_test_variable_first_submission') > 0.5 else 'B',
     }
+
+
+def get_hubspot_fields_tracked_on_registration(request, username):
+    body_unicode = request.body.decode('utf-8')
+    data = json.loads(body_unicode)['data']
```

Comment:
If this is post data it should be available in `request.POST`, which has the same interace as a dictionary but you won't have to decode and parse the raw body

Reply:
Hi, @calellowitz I've checked that and unfortunately the `dict` in the POST ie empty. I've tried several times to get the data form request.POST but with no luck, maybe there is sth I don't know about requests in python. 
![image](https://user-images.githubusercontent.com/18209901/39291775-64309f46-4934-11e8-82ec-4075dc253ce9.png)